tests/git-scripts/pre-push.sh

```diff
@@ -32,18 +32,9 @@ echo "URL is $URL"
 
 set -eu
 
-run_test()
-{
-    TEST=$1
-    echo "running '$TEST'"
-    if ! `$TEST > /dev/null 2>&1`; then
-        echo "test '$TEST' failed"
-        return 1
-    fi
+# shorten output but be sure to catch any error
+not() {
+    ! "$@"
 }
-
-run_test ./tests/scripts/check-doxy-blocks.pl
-run_test ./tests/scripts/check-names.sh
-run_test ./tests/scripts/check-generated-files.sh
-run_test ./tests/scripts/check-files.py
-run_test ./tests/scripts/doxygen.sh
+{ tests/scripts/all.sh -k 'check_*' || echo "FAILED all.sh"; } 2>&1 |
```

Comment:
Note that this also runs `check_python_files`, which we don't run on the CI because it is quietly skipped if `pylint3` is not available and our CI doesn't have `pylint3`. Since we don't run `check_python_files` on CI, it tends to get broken; it's currently broken on both `mbed-crypto/development` and `mbedtls/development`.

Reply:
You fixed that in the meantime, so marking as resolved.